### ISSUE TITLE
What is the status of this project (in relation to MiniScaffold)?

### ISSUE THREAD

**Issue by halcwb** [fsprojects/ProjectScaffold#352]
### Description

I have used this setup in the past. However, it seems to me that this project is no longer actively maintained. For example, the last commit is about a year ago, and the CI status is showing in red.

There is, however, an alternative: [https://github.com/TheAngryByrd/MiniScaffold](https://github.com/TheAngryByrd/MiniScaffold.)

Wouldn't it be better to either deprecate this project in favor of the 'new' MiniScaffold? Or is there still a use case for having these 2 similar projects living alongside. Seems to me that this will only be confusing for F# newbies?

### Repro steps

NA

### Expected behavior

NA

### Actual behavior

NA

### Known workarounds

NA

### Related information 

NA

**Comment by forki** [comment]
Yes right now Miniscaffold or the SAFE template are the things I'd
recommend.

Casper <notifications@github.com> schrieb am Do., 20. Feb. 2020, 22:52:

**Comment by wallymathieu** [comment]
Perhaps time to archive this project then? @forki
